### DISCUSSION TITLE
[Remix] Errors using per-route Edge functions on Remix 1.19

### DISCUSSION THREAD

**Discussion by thedambar** [vercel/vercel#10402] [Help]
### Question

Hope this is the right place to post this.  I've been working on a Remix project, but ran into issues when trying to make one of my routes an edge function following the instructions [here](https://vercel.com/blog/vercel-remix-integration-with-edge-functions-support).

After some failed attempts to debug the issue, I tried to revert to the most basic application possible with a new Remix install.  Specifically, I used:

`npx create-remix@latest`

I selected "Just the basics" and Vercel as the deploy target.  Then, I created a new route, edge.tsx:

```
export const config = {runtime: 'edge'};

export default function Edge() {
  <div>
    Edge
  </div>
}
```

When I try to deploy, I get the following error:

```
Error: The Edge Function "edge" is referencing unsupported modules:
--
15:11:21.708 | - api/build-edge-eyJydW50aW1lIjoiZWRnZSJ9.js: node:stream
15:11:21.708 | - @remix-run: crypto, fs, path, stream, fs/promises, os
15:11:21.708 | - cookie-signature: crypto
15:11:21.708 | - source-map-support: path
15:11:21.708 | - stream-slice: stream
```

Incidentally, these are the same issues I'm encountering in my main project.  

Any idea why this is happening?  Is the per-route edge functionality limited to older versions of Remix?  Am I doing something else wrong?

I found a similar discussion [here](https://github.com/vercel/vercel/discussions/9537), and so tried removing the only import from @remix-run/node that was in the base file.

**Comment by cill-i-am** [comment]
I think `npx create-remix@latest` references an old, unsupported version of the `@remix-run/vercel` package. Use this as a template and let me know how you get on: https://github.com/vercel-labs/remix-on-the-edge

It uses @vercel/remix which is the package that supports edge deploys
